dds_web/api/user.py

```diff
@@ -74,27 +74,27 @@ class ShowUsage(flask_restful.Resource):
     def get(self):
         current_user = auth.current_user()
 
-        # Check that user is facility account
-        if current_user.role != "facility":
+        # Check that user is unit account
+        if current_user.role != "unit":
             flask.make_response(
-                "Access denied - only facility accounts can get invoicing information.", 401
+                "Access denied - only unit accounts can get invoicing information.", 401
             )
 
-        # Get facility info from table (incl safespring proj name)
+        # Get unit info from table (incl safespring proj name)
         try:
-            facility_info = models.Facility.query.filter(
-                models.Facility.id == func.binary(current_user.facility_id)
+            unit_info = models.Unit.query.filter(
+                models.Unit.id == func.binary(current_user.unit_id)
             ).first()
         except sqlalchemy.exc.SQLAlchemyError as err:
-            return flask.make_response(f"Failed getting facility information: {err}", 500)
+            return flask.make_response(f"Failed getting unit information: {err}", 500)
```

Comment:
It complains about information exposure. Shall we log the err and take it out of the make_response message?

Reply:
Yes, we can do that.